src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,41 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName))) {
+            String line;
+            String[] temporaryArray;
+            int supplyInt = 0;
+            int buyInt = 0;
```

Comment:
The variable names 'supplyInt' and 'buyInt' could be more informative. Consider renaming them to 'totalSupply' and 'totalBuy' to better reflect their purpose.